x-pack/plugin/eql/qa/common/src/main/resources/test_queries_supported.toml

```diff
@@ -17,3 +17,26 @@ file where between(file_path, "dev", ".json", true) == "\\TestLogs\\something"
 [[queries]]
 query = 'process where string(serial_event_id) = "1"'
 expected_event_ids  = [1]
+
+[[queries]]
+expected_event_ids  = [98]
+notes = "regexp doesn't support character classes"
+query = '''
+//
+//                                ?'.*?net1\s+localgroup.*?')
+process where match(command_line, ?'.*?net1[ ]+localgroup.*?')
+'''
+
+[[queries]]
+expected_event_ids  = [98]
+query = '''
+process where match(command_line, ?'.*?net1.*?')
+'''
+
+[[queries]]
+expected_event_ids  = [98]
+notes = "regexp doesn't support character classes"
+query = '''
+//                                ?'.*?net1\s+\w{4,15}\s+.*?'
+process where match(command_line, ?'.*?net1[ ]+[a-z]{4,15}[ ]+.*?')
+'''
```

Comment:
There is no test here with more than one pattern. Can you add some, please?

Reply:
added in 521b3f9d137